docs/DeveloperGuide.md

```diff
@@ -217,6 +217,26 @@ Given below is the sequence diagram of command alias feature (This change only r
 
 ![aliasSequenceDiagram](images/commandAlias/aliasSequenceDiagram.png)
 
+#### Handling storage for *AliasMap*
+
+ **1. Model component**
+
+* `ModelManager` contains a `AliasMap`
+
+* `Model#updateAlias(Map<String, ValidCommand> map)` update the model with a `Map`
```

Comment:
Typo: `Model#updateAlias(Map<String, ValidCommand> map)` **updates** the model with a `Map`